packages/launchpad/cypress/e2e/project-setup.cy.ts

```diff
@@ -299,13 +299,13 @@ describe('Launchpad: Setup Project', () => {
 
         verifyWelcomePage({ e2eIsConfigured: false, ctIsConfigured: false })
 
-        cy.tabUntil((el) => el.text().includes('E2E Testing'))
+        cy.tabUntil((el) => el.text().includes('E2E testing'))
```

Comment:
The following suggestions correspond to `E2E Testing` and `Component Testing` on this screen

<img width="500" alt="Screen Shot 2022-09-29 at 7 44 15 PM" src="https://user-images.githubusercontent.com/26726429/193179007-f02c0365-7ee5-486b-8268-641be5dfbb90.png">

```suggestion
        cy.tabUntil((el) => el.text().includes('E2E Testing'))
```